src/envs/behavior_options.py

```diff
@@ -602,15 +651,15 @@ def grasp_obj_at_pos(
 
     # If the object is too big to be grasped, or bolted to its surface,
     # fail and return None
-    if (volume < 0.3 * 0.3 * 0.3 and
+    if not (volume < 0.3 * 0.3 * 0.3 and
```

Comment:
lol was this a bug before?

Reply:
yup, and it's why grasping wasn't working